isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/spring.adoc

```diff
@@ -120,6 +118,40 @@ public class CoreConfig {
 Die Spring-Beans der Komponenten werden mit der Annotation `@Component` bzw. deren Spezialisierungen (`@Repository`, `@Service`, `@Controller` etc.) versehen.
 Externe Spring-Beans, die nicht annotiert werden können, werden in der Konfigurationsklasse der Schicht per `@Bean`-Methoden konfiguriert.
 
+[[Empfehlungen-zu-Konfigurationsmoeglichkeiten-der-Applikationsklasse]]
```

Comment:
Dieser Inhalt wird jetzt hintenangestellt. Dadurch wird z.B. Component Scanning erst erklärt, nachdem ein paar Mal auf diese Funktion eingegangen wird. Es wäre vielleicht besser, den Inhalt in seiner Reihenfolge zu belassen.

Reply:
Die Reihenfolge wurde angepasst.